### PR TITLE
About Page

### PR DESCRIPTION
**Pull request by luisgov**
This PR adds the about page that will potentially be used as a home page. This informs the user how JEDI works and the steps needed to get started.

screencapture-localhost-8000-about-2018-10-02-11_46_43

### CODE REVIEW

**Comment by patricksmithdds** [comment]
@luiscielak can you run the formatter (`./script/format`) to fix the build?

File: styles/components/_banner.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.site-banner .usa-banner-inner {
+  max-width: 100%;
+}
+.site-banner .usa-banner-content {
+  margin-left: 0;
+}
```
Comment (patricksmithdds) [inline styles/components/_banner.scss]: Are these `site-banner` changes relevant or should they be part of #366?

File: templates/about.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      <a class="usa-button" href="{{ url_for('requests.requests_form_new', screen=1) }}"><span>New JEDI Request</span></a>
+
+
+      <h2>Five Steps to the JEDI Cloud</h2>
+      <p>Approval to use JEDI can be granted in hours once a task order has been created and financial verification provided. Follow the five steps below:</p>
+
+      <div class="home-container__list">
+        <ol>
+          <li>
+            <h3 class="h4">Complete a JEDI Cloud access request</h3>
```
Comment (patricksmithdds) [inline templates/about.html]: The "Cloud access request" does not have consistent capitalization with the text below. Should the "A" and "R" of "access request" be capitalized everywhere?

File: styles/components/_banner.scss (hunk 1 above)
Comment (luisgov) [reply styles/components/_banner.scss]: Removed from this branch.

File: templates/about.html (hunk 2 above)
Comment (luisgov) [reply templates/about.html]: Update capitalization to keep everything consistent.

**Pull request merged**